### ISSUE TITLE
Multiple File Upload from Single Input

### ISSUE THREAD

**Issue by bpiv400** [richardgirges/express-fileupload#58]
Hi, the documentation indicates express-fileupload supports multiple file upload when the files are coming from input tags with different names. But does the package support uploading multiple files from a single input tag with the multiple attribute? Something like this:
`<input type="file" name="myPictures" multiple></input>`

**Comment by richardgirges** [comment]
Hi @bpiv400 I believe this should work, although I haven't tested it myself. Are you seeing behavior to indicate otherwise?

**Comment by MarcProe** [comment]
It works, but the names of the uploaded files are a bit messy:

```
{ 'file[0]': 
   { name: 'file1.pdf',
     data: <Buffer 11 22 33 ... >,
     encoding: '7bit',
     mimetype: 'application/pdf',
     mv: [Function: mv] },
  'file[1]': 
   { name: 'file2.pdf',
     data: <Buffer 99 88 77 66  ... >,
     encoding: '7bit',
     mimetype: 'application/pdf',
     mv: [Function: mv] } }
```

Multiple fileupload will present the data as array to the server, and express-fileupload seems to not understand that and just take the array elements literally.

**Comment by richardgirges** [comment]
@MarcProe  there is another bug around issue of the object structure: #51 

I 100% agree that we need to fix the messiness of the uploaded file structure. I'll be tracking the progress for that fix in #51 and close this ticket in the meantime

**Comment by Suley** [comment]
> @MarcProe there is another bug around issue of the object structure: #51
> 
> I 100% agree that we need to fix the messiness of the uploaded file structure. I'll be tracking the progress for that fix in #51 and close this ticket in the meantime

Have you fixed it yet? Thanks.
